### DISCUSSION TITLE
Generating interface from wsdl

### DISCUSSION THREAD

**Discussion by westfin** [CoreWCF/CoreWCF#900] [Q&A] [answered]
Hey guys, is there any way to generate C# interface from wsdl which will be consumed by ASP.NET Core+ CoreWCF?

Some context: we have legacy soap-based service which had written on C++, and we want to rewrite to new asp.net core. But there are tons different methods and classes

**Comment by g7ed6e** [comment, marked as answer]
Hello @westfin I think you should give a try to [svcutil](https://learn.microsoft.com/en-us/dotnet/framework/wcf/servicemodel-metadata-utility-tool-svcutil-exe) using the /sc option.
